core/templates/dev/head/dashboard/dashboard.html

```diff
@@ -117,20 +129,56 @@ <h1 class="stat-value"><[dashboardStats.total_open_feedback]></h1>
                  class="oppia-dashboard-list-view-item">
           <table class="oppia-dashboard-table">
              <colgroup>
-               <col style="width: 30%;">
-               <col style="width: 14%;">
-               <col style="width: 14%;">
-               <col style="width: 14%;">
-               <col style="width: 14%;">
-               <col style="width: 14%;">
+               <col style="width: 26%;">
+               <col style="width: 13%;">
+               <col style="width: 13%;">
+               <col style="width: 16%;">
+               <col style="width: 16%;">
+               <col style="width: 16%;">
             </colgroup>
             <tr>
-              <th>Exploration</th>
-              <th>Rating</th>
-              <th>Plays</th>
-              <th>Feedback</th>
-              <th>Unresolved Answers</th>
-              <th>Last Updated</th>
+              <th ng-click="sortExplorationsList(EXPLORATIONS_SORT_BY_KEYS.TITLE)">
+                Exploration
+                <span ng-if="currentSortType === EXPLORATIONS_SORT_BY_KEYS.TITLE"
+                      class="fa"
+                      ng-class="isCurrentSortDescending ? 'fa-caret-up': 'fa-caret-down'">
```

Comment:
I avoided repetition but didn't put up a function on `$scope` since it doesn't seem to be needed now.